### DISCUSSION TITLE
Does this work with Lucene full text search?

### DISCUSSION THREAD

**Discussion by bsara** [afs/rdf-delta#52] [Q&A] [answered]
Hello, I've been trying to get rdf-delta working with Fuseki servers that use full text indexing (with the Jena provided text indexing functionalities using Lucene) with very little luck so far. Is there an example of how one could get this working or should I even expect it to work?

Thanks.

**Comment by afs** [comment, marked as answer]
Hi there,

rdf-delta could work with Fuseki+Lucene - the stack of datasets would be `changes dataset > text indexing > RDF storage`. The patch and changes layer guarantees that all changes are applied. In extreme (multiple machine failures between first problem and full recovery), there is some replay, which the text indexing should be oblivious to.

The barrier at the moment is simply that the text indexing isn't packed into the Fuseki server jar file. I don't see why if can't be added. 

Recorded as #53.

**Reply by bsara** [reply]
Thank you so much! I already had the stack of datasets set up correctly, I just needed the text indexing jar it seems.
